modules/tinymce/src/models/dom/main/ts/table/api/Clipboard.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+/*
+ NOTE: This file is duplicated in the following locations:
+  - plugins/table/api/Clipboard.ts
+ Make sure that if making changes to this file, the other files are updated as well
+ */
+
+import { Arr, Optional } from '@ephox/katamari';
+import { SugarElement } from '@ephox/sugar';
+
+import FakeClipboard from 'tinymce/core/api/FakeClipboard';
+
+type RowElement = HTMLTableRowElement | HTMLTableColElement;
+
+const tableTypeBase = 'x-dom/mce-table-';
```

Comment:
```suggestion
const tableTypeBase = 'x-tinymce/dom-table-';
```
Maybe something like this instead? I'm just trying to keep this consistent with the paste mime type (`x-tinymce/html`) and to also make the first part always be unique/namespaced to TinyMCE.

Reply:
That sounds reasonable to me 👍

Reply:
Done for both files